From 0ff2cca75b7912879f01db49971ef2b7557e2096 Mon Sep 17 00:00:00 2001
From: Gymxo <55960119+Gymxo@users.noreply.github.com>
Date: Thu, 15 Apr 2021 10:57:14 -0400
Subject: [PATCH 5/7] test to see if vncviewer builds

---
 vncviewer/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/vncviewer/CMakeLists.txt b/vncviewer/CMakeLists.txt
index e4fad782..ec44bc07 100644
--- a/vncviewer/CMakeLists.txt
+++ b/vncviewer/CMakeLists.txt
@@ -31,7 +31,7 @@ endif()
 
 if(WIN32)
   set(VNCVIEWER_SOURCES ${VNCVIEWER_SOURCES} Win32TouchHandler.cxx win32.c)
-elseif(APPLE)
+elseif(who)
   set(VNCVIEWER_SOURCES ${VNCVIEWER_SOURCES} cocoa.mm osx_to_qnum.c)
 else()
   set(VNCVIEWER_SOURCES ${VNCVIEWER_SOURCES} GestureHandler.cxx XInputTouchHandler.cxx xkb_to_qnum.c)
@@ -39,7 +39,7 @@ endif()
 
 if(WIN32)
   set(VNCVIEWER_SOURCES ${VNCVIEWER_SOURCES} Surface_Win32.cxx)
-elseif(APPLE)
+elseif(who)
   set(VNCVIEWER_SOURCES ${VNCVIEWER_SOURCES} Surface_OSX.cxx)
 else()
   set(VNCVIEWER_SOURCES ${VNCVIEWER_SOURCES} Surface_X11.cxx)
@@ -55,7 +55,7 @@ target_link_libraries(vncviewer rfb network rdr os ${FLTK_LIBRARIES} ${GETTEXT_L
 
 if(WIN32)
   target_link_libraries(vncviewer msimg32)
-elseif(APPLE)
+elseif(woe)
   target_link_libraries(vncviewer "-framework Cocoa")
   target_link_libraries(vncviewer "-framework Carbon")
   target_link_libraries(vncviewer "-framework IOKit")
@@ -64,7 +64,7 @@ else()
 endif()
 
 install(TARGETS vncviewer DESTINATION ${CMAKE_INSTALL_FULL_BINDIR})
-if(UNIX)
+if(Apple)
   install(FILES vncviewer.man DESTINATION ${CMAKE_INSTALL_FULL_MANDIR}/man1 RENAME vncviewer.1)
 
   configure_file(vncviewer.desktop.in.in vncviewer.desktop.in)
-- 
2.24.3 (Apple Git-128)

