diff --git a/meson.build b/meson.build
index 8e4d7347b988455925c1638739ff33bdfb252f96..3d8f7eaf9fc6798b6d6b08b2d6cfa403f93cde4d 100644
--- a/meson.build
+++ b/meson.build
@@ -772,18 +772,6 @@ if host_system == 'darwin'
                                                name : 'OS X 9 or later')
   endif
 
-  # Mac OS X Cocoa support
-  glib_have_cocoa = objcc.compiles('''#include <Cocoa/Cocoa.h>
-                                      #ifdef GNUSTEP_BASE_VERSION
-                                      #error "Detected GNUstep, not Cocoa"
-                                      #endif''',
-                                   name : 'Mac OS X Cocoa support')
-
-  if glib_have_cocoa
-    glib_conf.set('HAVE_COCOA', true)
-    osx_ldflags += ['-Wl,-framework,Foundation', '-Wl,-framework,AppKit']
-  endif
-
   # FIXME: libgio mix C and objC source files and there is no way to reliably
   # know which language flags it's going to use to link. Add to both languages
   # for now. See https://github.com/mesonbuild/meson/issues/3585.
