From 8f17c4df25dfc820ba24b18bc5575282dd2e5f22 Mon Sep 17 00:00:00 2001
From: Gymxo <55960119+Gymxo@users.noreply.github.com>
Date: Thu, 8 Apr 2021 10:38:55 -0400
Subject: [PATCH 13/15] remove os x

---
 src/linuxfb.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/linuxfb.c b/src/linuxfb.c
index e1c028b..04aa0ef 100644
--- a/src/linuxfb.c
+++ b/src/linuxfb.c
@@ -41,7 +41,6 @@ so, delete this exception statement from your version.
 #include "allowed_input_t.h"
 #include "uinput.h"
 #include "keyboard.h"
-#include "macosx.h"
 
 #if HAVE_SYS_IOCTL_H
 #include <sys/ioctl.h>
@@ -84,10 +83,6 @@ char *console_guess(char *str, int *fd) {
 	char *atparms = NULL, *file = NULL;
 	int do_input, have_uinput, tty = -1;
 
-#ifdef MACOSX
-	return macosx_console_guess(str, fd);
-#endif
-
 
 	if (strstr(in, "/dev/fb") == in) {
 		free(in);
-- 
2.24.3 (Apple Git-128)

